Normalizer/CustomEntityNormalizer.php

```diff
@@ -44,9 +44,13 @@ public function normalize($entity, $format = null, array $context = []): array
         $normalizedEntity = $this->pimSerializer->normalize($entity, 'standard', $context);
 
         $meta = [
-            'id'               => $entity->getId(),
-            'customEntityName' => $context['customEntityName'],
-            'form'             => $context['form'],
+            // Seems the upstream pim introduced some verioning for the javascript 
+            // assets. Not sure how to implement this correct and dont have the time
+            // to investigate this now. Hardcoding the version for now
```

Comment:
I had a discussion with my mates from the Core and they should make a PR to re-enable easy versioning.
Good catch, it remembers me to check the Core
:+1:

Reply:
I found the PR this morning: https://github.com/akeneo/pim-community-dev/pull/7133
I'll study what is the impact